client/me/notification-settings/main.jsx

```diff
@@ -18,11 +18,12 @@ import store from 'lib/notification-settings-store';
 import { fetchSettings, toggle, saveSettings } from 'lib/notification-settings-store/actions';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import PushNotificationSettings from 'components/push-notification/settings';
```

Comment:
That is correct. I moved the settings there, did the necessary renaming, and split up the scss.